tests/disjoint-union-perf/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -84,13 +84,14 @@ export type TypedNode =
  * @flow
  */
 
-export type InferredType = \"unknown\" |
-  \"gender\" |
-  \"enum\" |
-  \"number-or-string\" |
-  \"number\" |
-  \"string\" |
-  \"error\";
+export type InferredType = 
```

Comment:
Hm, I'm not sure. I think I'm going to revisit declarations again soon, so we'll figure it out.